### ISSUE TITLE
[Error] distrobox-create takes infinite time if gvfs volume is mounted

### ISSUE THREAD

**Issue by alcir** [89luca89/distrobox#47]
`distrobox-create` at some point perform
`+++ find /run -iname '*sock' '!' -path '/run/user/*'`

If a remote resource, like i.e. Nextcloud or GDrive or an SFT resource is mounted through Nautilus (`/run/user/1000/gvfs/`), the task could take a lot (probably the infinity) of time. Sometimes then, the create process seems stuck.

Indeed, if you run
`find /run -iname '*sock' '!' -path '/run/user/*'`
And for instance you have an sftp resource containing 150 files mounted, such find command will never end (after 30 minutes I interrupted it) with a lot of messages like
```text
find: ‘/run/user/1000/gvfs/sftp:host=fedorapeople.org,user=xxxx/dev/vga_arbiter’: Input/output error
find: ‘/run/user/1000/gvfs/sftp:host=fedorapeople.org,user=xxxx/proc/tty/driver’: Permission denied
```

**Comment by 89luca89** [comment]
Hey @alcir thanks a lot for this report, I'll double check and test some combinations of GoA and sftp :+1:

**Comment by 89luca89** [comment]
Tested and confirmed, the problem is that `find` will still enter and search an excluded path, we need to prune it.

Changed the command to:

`find /run -name 'user' -prune -o -name *sock* -print 2>/dev/null`

and it works as expected. Opened a PR for the fix, if you can test it and confirm it works also for you I'll merge and the fix will be included in the next release :+1:

**Comment by alcir** [comment]
I confirm that it works.

**Comment by 89luca89** [comment]
Cool :+1: fix will be included in 1.2.5 release
